src/theme/main.less

```diff
@@ -1,20 +1,62 @@
-// Separator Block
+// //Separator Block
 .block.separator {
+  top: unset;
   display: block;
   height: 54px;
```

Comment:
I think we should unset the height here (to override the one inherited from somewhere else) and let the height be derived automatically from the content and padding.